### DISCUSSION TITLE
manager#github-actions Enterprise Support

### DISCUSSION THREAD

**Discussion by ghost** [renovatebot/renovate#11998] [Request Help] [answered]
## Screenshot
![image](https://user-images.githubusercontent.com/50320942/135809550-872a3c36-2b9b-48a5-894d-1672c61b6993.png)

## Question
When using GHES (3.0 for now) you shouldn't/can't use _public_ actions. Thus the actions are in a specific organization following the naming:

```
owner/repo@tag --> org/owner-repo@tag
```

It would therefor be necessary to tell the manager that he should compare towards another baseUrl. I can not see that option in the docs https://docs.renovatebot.com/modules/manager/github-actions/.

Does anyone have an idea if its just not documented or not yet supported?

**Comment by Shegox** [comment, marked as answer]
We figured it out. You can adjust the datasource for the `github-tags` to point to your Enterprise Server. Adding following config to your repository or bot configuration should do the trick: 
```json
    "packageRules": [
    {
      "matchDatasources": ["github-tags"],
      "matchManagers": ["github-actions"],
      "registryUrls": ["https://github.enterprise.com"]
    }
  ]
 ```
 
 If you have your GHES configured to use github.com actions as fallback following should work adjusting the priority of the `registryUrls` should work like this: 
```json
"registryUrls": ["https://github.enterprise.com", "github.com"]
```

It might make sense to adjust Renovate to use this pattern as the default if it notices that it runs against GHES.
